src/main/java/uk/gov/hmcts/sscs/service/AuthorisationService.java

```diff
@@ -28,9 +29,9 @@ public Boolean authorise(String serviceAuthHeader) {
         } catch (FeignException exc) {
             boolean isClientError = exc.status() >= 400 && exc.status() <= 499;
 
-            LOG.error("Authorisation failed for Notification request with status: " + exc.status(), exc);
+            LOG.error("Authorisation failed for Notification request with status: {}", exc.status(), exc);
 
-            throw isClientError ? new AuthorisationException(exc.getMessage(), exc) : exc;
+                throw isClientError ? new ClientAuthorisationException(exc) : new AuthorisationException(exc);
```

Comment:
There are two minor updates I would suggest:

- logger does not have method with arguments as follows: message, arguments, exception. must rewrite to `log.error(String.format(message, arguments), exception)` to correctly represent log event
- avoid alert levels by including your own exceptions in logger. create exception first (from line 34), use it in logger (line 32) then throw (afterwords)